Controller/ResourceController.php

```diff
@@ -32,21 +34,34 @@ public function __construct(
 
     public function resourceAction($repositoryName, $path)
     {
-        $repository = $this->registry->get($repositoryName);
-        $resource = $repository->get('/' . $path);
-
-        $context = SerializationContext::create();
-        $context->enableMaxDepthChecks();
-        $context->setSerializeNull(true);
-        $json = $this->serializer->serialize(
-            $resource,
-            'json',
-            $context
-        );
-
-        $response = new Response($json);
-        $response->headers->set('Content-Type', 'application/json');
-        
+        try {
+            $repository = $this->registry->get($repositoryName);
+            $resource = $repository->get('/' . $path);
+
+            $context = SerializationContext::create();
+            $context->enableMaxDepthChecks();
+            $context->setSerializeNull(true);
+            $json = $this->serializer->serialize(
+                $resource,
+                'json',
+                $context
+            );
+
+            $response = new Response($json);
+            $response->headers->set('Content-Type', 'application/json');
+        } catch (\Exception $e) {
```

Comment:
this seems problematic. if there is an error handler in the project, it will not work as we start hiding the exception from symfony

maybe we should instead provide an error handler that would convert the exception to json